src/first/Main.java

```diff
@@ -15,11 +15,17 @@
     public static void main(String[] args) throws Exception {
```

Comment:
comment to 
static Scanner scanner = new Scanner(System.in);

its time to use them.